File: frontend/src/app/master/models/sla.ts
export interface SLA {
  slaId: number;
  customerId: string;
  customerName: string;
  projectId: string;
  projectName: string;
  priority: string;
  slaHours: number;
  remarks?: string;  
  active: boolean;
  status?: number;
  createdOn?: string;    
  createdBy?: string;
  modifiedOn?: string;
  modifiedBy?: string;
}

File: backend/Models/SLA.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace backend.Models
{
    [Table("SLA")]
    public class SLA:BaseModel
    {
        [Key]
        public long SLAId { get; set; }
        public string CustomerId {  get; set; }
        public string CustomerName { get; set; }
        public string ProjectId {  get; set; }
        public string ProjectName { get; set; }
        public string Priority { get; set; }
        public double SLAHours { get; set; }
        public string? Remarks { get; set; }

    }

    [Table("Response_SLA")]
    public class ResponseSLA : BaseModel
    {
        [Key]
        public long Id { get; set; }
        public string CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string ProjectId { get; set; }
        public string ProjectName { get; set; }
        public int PriorityId { get; set; }
        public double SLAHours { get; set; }
        public string? Remarks { get; set; }

    }
}
